main.tf

```
@@ -22,6 +22,8 @@ locals {
     actions   = ["kms:Decrypt"]
     resources = [var.kms_key_arn]
   }
+
+  lambda_handler = split(".", basename(var.lambda_source_path))[0]
```

```suggestion
  lambda_handler = try(split(".", basename(var.lambda_source_path))[0], "notify_slack")
```

`var.lambda_source_path` can be `null`